functions_template

```diff
@@ -0,0 +1,28 @@
+function %TODO%() {
```

Comment:
Is this function valuable to leave in? Any future tools probably aren't going to be generalized with the same calling conventions as Terraform.

Reply:
The function name would map 1:1 with the name of the tool in use.  The following 3 `%TODO%`s allow you to specify the security checking tool that maps to the tool, the arguments to that tool, and then a human-readable explanation of what the security tool is attempting to do, respectively.  For instance, we may want to scan `ansible` code in the future, and I could envision this being useful with little to no modification in that case